app/src/main/res/values-pt/strings-pt.xml

```diff
@@ -1,496 +1,521 @@
+    <string name="deel_settings_for_algs">Definições de algoritmos</string>
+    <string name="low_level_prediction_values">Valores de previsão de Hipo</string>
+    <string name="glucose_drop_for_one_unit">Baixa de Glicémia por 1 uni.</string>
+    <string name="xdrip_plus_prediction_settings">Simulação Preditiva xDrip+</string>
+    <string name="settings_for_syncing">Definições para sincronização entre dispositivos</string>
+    <string name="xdrip_plus_sync_settings">Definições de sincronização xDrip+</string>
+    <string name="xdrip_plus_update_settings">Definições de atualizações xDrip+</string>
+    <string name="automatic_update_check">Verificar por atualizações automaticamente</string>
+    <string name="update_channel">Canal de Atualizações</string>
+    <string name="send_crash_errors_to_developer">Enviar relatório de erros para o programador</string>
+    <string name="automatic_crash_reporting">Report de erros automático</string>
+    <string name="send_feedback_message">Enviar Mensagem de Feedback</string>
+    <string name="copying_settings">Definições de Cópia</string>
+    <string name="show_settings_qr_codes">Mostrar códigos QR de Definições</string>
+    <string name="load_save_settings_to_sdcard">Carregar / Guardar definições no cartão SD</string>
+    <string name="alarms_and_alerts">Alarmes e Alertas</string>
+    <string name="alerts_and_notifications">Alertas e Notificações</string>
+    <string name="glucose_level_alerts_list">Lista de Níveis Glicémios de Alerta</string>
+    <string name="glucose_alerts_settings">Definições de Alertas de Glicémia</string>
+    <string name="smart_snoozing">Snooze inteligente</string>
+    <string name="calibration_alerts">Alertas de Calibração</string>
+    <string name="missed_reading_alert">Alerta de falha de leitura</string>
+    <string name="other_alerts">Outros Alertas</string>
+    <string name="extra_alerts_xdrip_plus">Alertas Extra (xDrip+)</string>
+    <string name="persistent_high_alert">Alerta de Hiper persistente</string>
+    <string name="forecasted_low_alert">Alerta de previsão de Hipo</string>
+    <string name="extrapolate_data_to_try_to_predict_lows">Extrapolar dados para tentar prever Hipos</string>
+    <string name="alarm_at_forecasted_low_mins">Alarm at Forecasted Low mins</string>
+    <string name="other_xdrip_plus_alerts">Outros alertas xDrip+</string>
+    <string name="xdrip_plus_display_settings">Definições de visualização xDrip+</string>
+    <string name="display_customisations">Personalizações de visualização</string>
+    <string name="insulin_carb_ratios_etc_for_models">Insulina, carios de HC etc para modelos</string>
+    <string name="grams_of_carbohydrate_one_unit_covers">Gramas de HC que 1 Uni. cobre</string>
+    <string name="linear_model_carbs_absorbed_per_hour">Modelo linear de absorção de HC por hora</string>
+    <string name="carb_absorption_rate">Rácio de Absorção de HC</string>
```

Comment:
I have used "HC" in quite some translations as it abbreviates "Hidratos de Carbono" which is very long. "HC" is commonly used by physicians, nutritionist  and hence, patients. I vote for keeping "HC". You you want me to change it, please reply.